### ISSUE TITLE
GDM: change sleep-inactive-ac-type default value from 'suspend' to 'nothing'

### ISSUE THREAD

**Issue by sebo505** [getsolus/packages#1834]
### Please confirm there isn't an existing open bug report

- [X] I have searched open bugs for this issue

### Summary

This is tightly related to [(T10537) #193](https://github.com/getsolus/packages/issues/193)

With GDM as login manager, if you don't login right away after boot, after a certain amount of inactivity, computer suspends automatically, regardless of users energy/suspend settings. For GDM, these settings can be changed via gesettings run as gdm user:
`su - gdm -s /bin/sh`
`dbus-launch --exit-with-session gsettings set org.gnome.settings-daemon.plugins.power sleep-inactive-ac-type 'nothing'`
or via the config file in /etc/dconf/db/gdm.d/ as described in the above mentioned issue.

My personal stance is that auto-suspend-on-inactivity should be deactivated per default in GDM, at least if you are on ac.
I understand the value of auto-suspend if you run on battery, but given that there are users for who wakeup after suspend does not work properly, for instance graphics not initalizing correctly, I would rather deactivate this.
You can still trigger suspend manually in GDM, and I think suspend is also still triggered if you close lid.

### Steps to reproduce

boot computer
on login-screen (GDM), don't login right away, wait like 6 min

### Expected result

suspend should not be triggered

### Actual result

suspend is triggered

### Environment

- [X] Is system up to date?

### Repo

Shannon (stable)

### Desktop Environment

GNOME

### System details

Hardware Information:

Hardware-Modell: Hewlett-Packard HP EliteBook 6930p
Speicher: 8,0 GiB
Prozessor: Intel® Core™2 Duo T9550 × 2
Grafik: AMD RV620
Festplattenkapazität: 250,1 GB

Software Information:

Firmware-Version: 68PCD Ver. F.16
Name des Betriebssystems: Solus 4.5 Resilience
Betriebssystem-Build: (null)
Betriebssystem-Typ: 64-bit
GNOME-Version: 45.3
Fenstermanager: X11
Kernel-Version: Linux 6.6.12-271.current

**Comment by sebo505** [comment]
Just to give a further explanation why I think current GDM behaviour is no good: I think for example of my parents, 70 and 80 years old. Scenario is the following: They start computer, then, while it's booting, they decide to go make a tea, or to check washing machine, or whatever, and then, when they return after a short while, before they even had the chance to login, computer is already in suspend and they don't understand whats going on.

**Comment by joebonrichie** [comment]
I agree, I can change the default in https://github.com/getsolus/gnome-desktop-branding when we land GNOME 46.

**Comment by sebo505** [comment]
That would be much appreciated.

**Comment by sebo505** [comment]
On a related note I'd like to mention that regardless of suspend beeing turned off for GDM with
`su - gdm -s /bin/sh`
`dbus-launch --exit-with-session gsettings set org.gnome.settings-daemon.plugins.power sleep-inactive-ac-type 'nothing'`
GDM screen will go blank after some time.
If screen blanking in GDM is undesired, this can be changed with
`su - gdm -s /bin/sh`
`dbus-launch --exit-with-session gsettings set org.gnome.desktop.session idle-delay '0'`

**Comment by ermo** [comment]
@joebonrichie Please do this ASAP for 4.6?

**Comment by TraceyC77** [comment]
This is not being included in 4.6, deferring to 4.7

**Comment by TraceyC77** [comment]
Joey and Reilly discussed this

**Comment by TraceyC77** [comment]
This is not a release blocker for 4.8, taking it out of the milestone. I'm also marking it as Low, since it's a requested change from upstream defaults that can be changed by users who need to. We've not gotten any other feedback on this.
